lib/lndclient/LndClient.ts

```diff
@@ -44,25 +41,16 @@ class LndClient {
       this.meta = new grpc.Metadata();
       this.meta.add('macaroon', adminMacaroon.toString('hex'));
 
-      this.setStatus(LndClient.statuses.CONNECTION_VERIFIED); // TODO: verify connection
+      this.setStatus(enums.statuses.CONNECTION_VERIFIED); // TODO: verify connection
     }
   }
 
-  isDisabled() {
-    return this.status === LndClient.statuses.DISABLED;
-  }
-
-  setStatus(val) {
-    this.logger.info(`lndClient status: ${val}`);
-    this.status = val;
-  }
-
   /**
    * Return general information concerning the lightning node including it’s identity pubkey,
    * alias, the chains it is connected to, and information concerning the number of
    * open+pending channels.
    */
-  getInfo() {
+  getInfo(): Promise<any> {
```

Comment:
You explicitly declare the return type. I think it's the better way. Let's stick to that, and do it also in `BaseClient`.